lms/djangoapps/course_api/views.py

```diff
@@ -193,17 +195,34 @@ class CourseListView(DeveloperErrorViewMixin, ListAPIView):
     pagination_class = NamespacedPageNumberPagination
     serializer_class = CourseSerializer
 
+    # Return all the results, 10K is the maximum allowed value for ElasticSearch.
+    # We should use 0 after upgrading to 1.1+:
+    #   - https://github.com/elastic/elasticsearch/commit/8b0a863d427b4ebcbcfb1dcd69c996c52e7ae05e
+    results_size_infinity = 10000
```

Comment:
Is this the number of results we are requesting? Are we paginating the results to the end user? I believe DRF handles the pagination, it is worth checking though.

Reply:
@staubina check the updated PR description for more details.